src/NUnitFramework/tests/Constraints/ToleranceTests.cs

```diff
@@ -111,5 +111,16 @@ public void ToStringTests()
                 Assert.That(Is.EqualTo(5).Within(2).Minutes.Tolerance.ToString(), Is.EqualTo("00:02:00"));
             });
         }
+
+        [Test]
+        public void EqualityTests()
+        {
+            var tol1 = Tolerance.Default;
+            var tol2 = Tolerance.Default;
+            Assert.That(tol1, Is.EqualTo(tol2));
```

Comment:
This could be an `Is.SameAs` test as there is only one `Default` and there is no way a user can create another instance as that constructor is `private`

Reply:
And that is already tested, see `TestToleranceDefaultIsSameAs`, so this test can be removed.